W-17050370-noteINpropertiesCH2-48-dm (#2841)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

  <http.port>8081</http.port>
</properties>
----

> If you need to redeploy the application and there aren't properties configured in the deployment POM file, the existing properties in your CloudHub 2.0 environment are kept. Note that if properties are defined in the POM, the app is deployed using those properties, overriding any properties set in the UI.

Reviewer: ```suggestion
If you must redeploy the application and there aren't properties configured in the deployment POM file, the existing properties in your CloudHub 2.0 environment are kept. Note that if properties are defined in the POM file, the app is deployed using those properties and overrides any properties set in the UI.
```
